Create vApp VM Security Tag Tests (#1046)
vmware/terraform-provider-vcd: website/docs/r/security_tag.html.markdown

Supported in provider *v3.7+* and requires VCD 10.3.0+

> ~> **Note:** Either `vcd_security_tag` resource or [`security_tags` attribute from `vcd_vapp_vm`](/providers/vmware/vcd/latest/docs/resources/vapp_vm)
> should be used as they would cause a behavioral conflict with each other.

Reviewer: ```suggestion
~> **Note:** Only one of `vcd_security_tag` resource or [`security_tags` attribute from `vcd_vapp_vm`](/providers/vmware/vcd/latest/docs/resources/vapp_vm)
should be used. Using both would cause a behavioral conflict.
```

Author: Updated
